### PR TITLE
Phase 3 update part 1

### PR DESCRIPTION
**Pull request by MrRubberDucky99**
I have worked on expanding the categories to give more choice as to what is shown as you suggested. I did not know how the icons you chose for your buttons worked as they seem to be hosted elsewhere, and so I just chose the TV design for the categories as that seemed to fit most. 
The added categories are:
- MCU TV
- Defenders TV
- Other TV
- Web Shows

I have also added a to-do list and a suggestions area to the read me that will cover both features that are currently being worked on and features that could be implemented in the future so people who want to contribute have a list of things they can work on. I have added phases to the to-do list as I plan on implementing it in the second part of this update>

I am relatively new to js so my code may not be the best (I mostly tried to compy and edit pre-existing code), but hopefully this is a good starting point.
Thanks

### CODE REVIEW

**Review by dispossible — CHANGES_REQUESTED** [review]
Hi Ducky,

First I want to give some context; this really isn't a full on open source project, it's public on GitHub really just for the use of the issue tracker to report problems to me. I coded this project in my spare time from my full time job doing web development as part of a development team. It's a project I do for fun where I have creative control and don't need to manage a large codebase or a team.

I make this point to highlight that I wasn't looking for others to be involved with this codebase. I don't really want to spend my Sunday evenings merging code and writing messages, when I could be writing some fun code myself. I update this project from time to time myself when the mood strikes me for a bit of fun and a break from other things (aside from maintaining the release data whenever needed).

I really appreciate your enthusiasm to get stuck in. You've done a bunch of work. I do need to just put some breaks on you though. I had picked up that you're just getting started with web development from your last pull request and not understanding my icons continues to show you're knowledge level. (Look up 'SVG graphics' on that topic btw)

I was hoping with this task to ease you over to helping out with the data side of this project, and not so much of the internal workings of the code. I would be happy to accept your changes to the data file to update the types, and the tweak to the parser function that reads the new types into the old 'tv' class to prevent errors.

I was not expecting you to implement the new types into filtering system, I have some ideas on a more fundamental refactoring of the way filtering works, but I needed the prerequisite of having the new types and the phase numbers in the data to filter on. The current system is fairly simplistic and was written at a time when there was vastly less shows. The code you have copied to expand on is adding more weight on that original weak foundation.

The rest of your code is technically good, it just suffers from just being more of my original simple system which is becoming overworked.

I do notably disapprove of the changes to the Readme file. I don't overly want to encourage contributors, so don't want to have the wording in there to that effect, I have no plans for community guidelines or anything of that ilk. I would also want to remove the suggestions section you added, I don't think it's the correct place in the project to store ideas that may or may not be implemented.

If you resubmit the pull request with just the changes to the data file and the utils.js I will accept it. Or do the same but with the phase numbers also added to the data.

tldr; I appreciate all your effort and interest but please back off a little bit, this is a very small side project I work on for fun. I'll accept your changes to the data file.

Thanks,
Timothy
